spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkMetadataColumns.java

```diff
@@ -317,28 +316,6 @@ public void testConflictingColumns() {
         sql("SELECT _spec_id, _partition, _renamed_spec_id FROM %s", TABLE_NAME));
   }
 
-  @TestTemplate
-  public void testRowLineageColumnsResolvedInV3OrHigher() {
```

Comment:
I'm removing this test now, it's not really useful anymore since any format version can project _row_id, and in the unsupported versions it'll be null.